keras_nlp/evaluations/glue/README.md

```diff
@@ -0,0 +1,96 @@
+# GLUE Finetuning Script
```

Comment:
Should we move this to `examples/` or `tools/`? It does seems like we may want to frame this as either example code to help you get going, or a literal tool for evaluating models we have checked in.

Reply:
I was thinking about that - my main concern is how we can increase the visibility of things not residing inside the package itself. 

Folding it under `examples/` seems to be a decent choice.